extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -207,11 +231,35 @@ Make payment request generated from Adyen Web Components for credit card payment
     {
       "action": "setCustomField",
       "name": "makePaymentRequest",
-      "value": "{\"amount\":{\"currency\":\"EUR\",\"value\":1000},\"reference\":\"YOUR_ORDER_REFERENCE\",\"paymentMethod\":{\"type\":\"scheme\",\"encryptedCardNumber\":\"test_4111111111111111\",\"encryptedExpiryMonth\":\"test_03\",\"encryptedExpiryYear\":\"test_2030\",\"encryptedSecurityCode\":\"test_737\"},\"returnUrl\":\"https://your-company.com/...\",\"merchantAccount\":\"YOUR_MERCHANT_ACCOUNT\"}"
+      "value": "{\"amount\":{\"currency\":\"EUR\",\"value\":1000},\"reference\":\"YOUR_REFERENCE\",\"paymentMethod\":{\"type\":\"scheme\",\"encryptedCardNumber\":\"test_4111111111111111\",\"encryptedExpiryMonth\":\"test_03\",\"encryptedExpiryYear\":\"test_2030\",\"encryptedSecurityCode\":\"test_737\"},\"returnUrl\":\"https://your-company.com/...\",\"merchantAccount\":\"YOUR_MERCHANT_ACCOUNT\"}"
     }
   ]
 }
 ```
+
+If you are [creating a new commercetools payment](https://docs.commercetools.com/http-api-projects-payments#create-a-payment), the payment draft have to contain the `paymentMethodInfo.paymentInterface = ctp-adyen-integration` and `amountPlanned` value additional to the `makePaymentRequest` custom field, for example:
```

Comment:
Yes, but I think it's ok to explain it again because this the first place in the integration that we are explaining a new payment draft. Is it ok to add only the link? Or what would be best?

Reply:
I agree, just a link is ok :)

Reply:
I think this is a different part, creating a new object is not related to the validation section, I could not rephrase the sentence with this validation link.

Reply:
```suggestion
If you are [creating a new commercetools payment](https://docs.commercetools.com/http-api-projects-payments#create-a-payment), the payment draft have to contain the `paymentMethodInfo.paymentInterface = ctp-adyen-integration` (see https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/docs/WebComponentsIntegrationGuide.md#validate-payment) and `amountPlanned` value additional to the `makePaymentRequest` custom field, for example:
```